tests/forte/data/data_store_test.py

```diff
@@ -245,23 +245,36 @@ def test_get(self):
         self.assertEqual(len(instances), 0)
 
     def test_delete_entry(self):
```

Comment:
could you add extra tests for types other than annotation? Just to make sure it is compatible with link and group.